src/proto/mgmt/pool.proto

```diff
@@ -28,20 +28,18 @@ message PoolCreateReq {
 	repeated uint32 faultDomains = 7; // Fault domain tree, minimal format
 	uint32 numsvcreps = 8; // desired number of pool service replicas
 	uint64 totalbytes = 9; // Total pool size in bytes (auto config)
-	double scmratio = 10; // Ratio of SCM:NVMe expressed as % (auto config)
+	repeated double tierratio = 10; // Ratio of storage tiers expressed as % of totalbytes (auto config)
```

Comment:
I seen now that the pool create request parameter names need updating to tier_ratio style notation

Reply:
https://jira.hpdd.intel.com/browse/DAOS-8014